core/src/apps/bitcoin/sign_tx/tx_info.py

```diff
@@ -108,7 +107,7 @@ def output_is_change(self, txo: TxOutput) -> bool:
             return False
         return (
             self.wallet_path.output_matches(txo)
-            and len(txo.address_n) >= BIP32_WALLET_DEPTH
+            and len(txo.address_n) >= common.BIP32_WALLET_DEPTH
```

Comment:
i'm surprised that there are two _additions_ of `common.FOO` here, and `common` stays a global import...?

Reply:
`Common` stays global and all functions call `common.XXX`, because there are many various `XXX` used only once, and therefore it is not space-wise-worth doing `from .common import input_is_external, CHANGE_OUTPUT_SCRIPT_TYPES, BIP32_WALLET_DEPTH`.

If `common.BIP32_WALLET_DEPTH` would be used many times, it would be beneficial to import it as a unique symbol.